gulpfile.js

```diff
@@ -66,6 +67,15 @@ gulp.task('js-libs', function () {
         .pipe(gulp.dest(path.join(output_directory, 'js')));
 });
 
+// Lints all js files
```

Comment:
Please move this task down to our extra tasks.

Reply:
I see that we already have a js-hint task. So please use (and maybe modify) our task.